modules/cli/pages/openshift-gateway.adoc

```diff
@@ -0,0 +1,446 @@
+The credentials for internal console-auth mode are held in secret: 'skupper-console-users'
+----
+--
+
+// Type: procedure
+[id="frontend"] 
+== Creating the frontend service
+
+The frontend service is a simple Python application that displays a message from the backend application.
+
+.Procedure
+
+Perform all tasks in the west terminal session:
+
+. Deploy the frontend service:
++
+[source,bash,term=west]
+----
+$ oc create deployment hello-world-frontend --image quay.io/pwright/hello-world-frontend
+----
+
+. Expose the frontend deployment as a cluster service:
++
+[source,bash,term=west]
+----
+$ oc expose deployment hello-world-frontend --port 8080 --type LoadBalancer
+----
+
+. Create a route for the frontend:
++
+[source,bash,term=west]
+----
+$ oc expose svc/hello-world-frontend
+----
+
+. Check the frontend route:
+.. Get the route details:
++
+--
+[source,bash,term=west]
+----
+$  oc get routes
+----
+
+The output should be similar to the following:
+
+----
+NAME                   HOST/PORT                                   
+hello-world-frontend   <frontend-url>       
+----
+--
+
+.. Navigate to the `<frontend-url>` value in your browser, you see a message similar to the following because the frontend cannot communicate with the backend yet:
++
+----
+Trouble! HTTPConnectionPool(host='hello-world-backend', port=8080): Max retries exceeded with url: /api/hello (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbfcdf0d1d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
+----
++
+To resolve this situation, you must create the backend service and make it available on the {service-network}.
+
+// Type: procedure
+[id="backend"] 
+== Creating the backend service and making it available on the {service-network}
+
+The backend service runs in the `east` namespace and is not available on the {service-network} by default.
+You use the `skupper` command to expose the service to all namespaces on the {service-network}.
+The backend app is a simple Python application that passes a message to the frontend application.
+
+.Procedure
+
+. Deploy the backend service in the east terminal session:
++
+[source,bash,term=east]
+----
+$ oc create deployment hello-world-backend --image quay.io/pwright/hello-world-backend
+----
+
+. Expose the backend service on the {service-network} from the east terminal session:
++
+[source,bash,term=east]
+----
+$ skupper expose deployment hello-world-backend --port 8080 --protocol tcp
+----
+
+. Check the site status from the west terminal session:
++
+--
+[source,bash,term=west]
+----
+$ skupper status
+----
+The output should be similar to the following:
+----
+Skupper is enabled for namespace "west" in interior mode. It is connected to 1 other site. It has 1 exposed service.
+----
+The service is exposed from the `east` namespace.
+--
+
+
+. Create a route for the backend:
++
+[source,bash,term=west]
+----
+$ oc expose svc/hello-world-backend
```

Comment:
For the example, is there a need to expose the backend service or can it remain on the skupper network only?

Reply:
removed the backend totally